### PR TITLE
PXB-3217 Add titles to the index page (Innovation)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/backup-overview.md
	modified:   docs/create-full-backup.md
	modified:   docs/index.md
	modified:   docs/restore-a-backup.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -29,4 +29,38 @@ Percona XtraBackup {{vers}} does not support making backups of databases
...
+Taking your backup with Percona XtraBackup is easy. Follow our documentation guides, and you’ll be set up in quickly.
+
+<div data-grid markdown><div data-banner markdown>
+
+## :material-frequently-asked-questions: How Percona Xtrabackup works { .title }
+
+Learn about how Percona Xtrabackup works.
+
+[How Percona XtraBackup works :material-arrow-right:](how-xtrabackup-works.md){.md-button}
+
```
Comment (nastena1606) [inline docs/index.md:41]: This one should be closed with </div><div data-banner markdown>

File: docs/index.md [hunk 2]
```diff
@@ -29,4 +29,38 @@ Percona XtraBackup {{vers}} does not support making backups of databases
...
+</div><div data-banner markdown>
+
+### :material-backup-restore: Backup management { .title }
+
+Learn about different types of backups and how to take them.
+
+[Backup management :material-arrow-right:](backup-overview.md){ .md-button }
+
+</div><div data-banner markdown>
```
Comment (nastena1606) [inline docs/index.md]: Remove data-banner markdown from last <div>

Comment (alina-derkach-oaza) [reply docs/index.md]: done

File: docs/index.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/index.md:41]: done

**Review by patrickbirch — APPROVED** [review]
LGTM

**Pull request merged**